### DISCUSSION TITLE
Performance benchmark

### DISCUSSION THREAD

**Discussion by Madara9744** [Madara9744/PasswordManagement#8] [Q&A] [answered]
@rohankumardubey Can you perform the benchmarks and let me see the stats?

**Comment by rohankumardubey** [comment, marked as answer]
yes made benchmarks for MapDb and Helidon found some statistics

MapDb 300 ops/sec
Helidon 125 ops/sec
was for the single-threaded use case. let me collect further info for concurrent reads.

**Reply by Madara9744** [reply]
thanks can look for other alternate solutions as well and perform benchmarks.
